de.tubs.variantsync.core/src/de/tubs/variantsync/core/patch/AMarkerHandler.java

```diff
@@ -24,6 +24,7 @@
 	public abstract List<IVariantSyncMarker> getMarkers(IFile file, int offset, int length);
 
 	@Override
-	public abstract boolean updateMarkerForDelta(SourceFile sourceFile, IDelta<T> delta, List<IVariantSyncMarker> variantSyncMarkers);
+	public abstract boolean updateMarkerForDelta(SourceFile sourceFile, IDelta<T> delta,
+			List<IVariantSyncMarker> variantSyncMarkers);
 
 }
```

Comment:
I do not get this `@Override` what is changed?

Reply:
This problem will be addressed by the following issue https://github.com/tthuem/VariantSync/issues/57